src/dsp_tools/commands/excel2json/properties.py

```diff
@@ -490,3 +492,11 @@ def excel2properties(
             print(f"properties section was created successfully and written to file '{path_to_output_file}'")
 
     return props, True
+
+
+def _read_check_property_df(excelfile: str) -> pd.DataFrame | None:
+    sheets_df_dict = read_and_clean_all_sheets(excelfile=excelfile)
+    if len(sheets_df_dict) != 1:
+        msg = str(MoreThanOneSheetProblem("properties.xlsx", list(sheets_df_dict.keys())))
+        raise InputError(msg)
+    return list(sheets_df_dict.values())[0]
```

Comment:
If a container is big and you want to return its first value, it's more performant to put it like this: `next(iter(sheets_df_dict.values()))`

But here, it doesn't matter, because it is always of length 1